docs/monetization/Entitlements.md

```diff
@@ -0,0 +1,184 @@
+## Entitlement Resource
+
+Entitlements in Discord represent that a user or guild has access to a premium offering in your application.
+
+### Entitlement Object
+
+###### Entitlement Structure
+
+| Field          | Type              | Description                                                                                 |
+|----------------|-------------------|---------------------------------------------------------------------------------------------|
+| id             | snowflake         | ID of the entitlement                                                                       |
+| sku_id         | snowflake         | ID of the SKU                                                                               |
+| user_id?       | snowflake         | ID of the user that is granted access to the entitlement's sku                              |
+| guild_id?      | snowflake         | ID of the guild that is granted access to the entitlement's sku                             |
+| application_id | snowflake         | ID of the parent application                                                                |
+| type           | integer           | [Type of entitlement](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object-entitlement-types) |
+| consumed       | boolean           | Not applicable for App Subscriptions. Subscriptions are not consumed and will be `false`    |
+| starts_at?     | ISO8601 timestamp | Start date at which the entitlement is valid. Not present when using test entitlements.     |
+| ends_at?       | ISO8601 timestamp | Date at which the entitlement is no longer valid. Not present when using test entitlements. |
+
+###### Entitlement Example
+
+```json
+{
+    "id": "1019653849998299136",
+    "sku_id": "1019475255913222144",
+    "application_id": "1019370614521200640",
+    "user_id": "771129655544643584",
+    "promotion_id": null,
+    "type": 8,
+    "deleted": false,
+    "gift_code_flags": 0,
```

Comment:
I see promotions and this gift code flag in the payload but not the docs. Are these used at all?